README.md

```diff
@@ -167,7 +169,7 @@ module "sf_loader" {
 | <a name="input_snowflake_aws_s3_bucket_name"></a> [snowflake\_aws\_s3\_bucket\_name](#input\_snowflake\_aws\_s3\_bucket\_name) | AWS bucket name where data to load is stored | `string` | n/a | yes |
 | <a name="input_snowflake_database"></a> [snowflake\_database](#input\_snowflake\_database) | Snowflake database name | `string` | n/a | yes |
 | <a name="input_snowflake_loader_user"></a> [snowflake\_loader\_user](#input\_snowflake\_loader\_user) | Snowflake username used by loader to perform loading | `string` | n/a | yes |
-| <a name="input_snowflake_password"></a> [snowflake\_password](#input\_snowflake\_password) | Password for snowflake\_loader\_user used by loader to perform loading | `string` | n/a | yes |
+| <a name="input_snowflake_password"></a> [snowflake\_password](#input\_snowflake\_password) | Snowflake password as a string or an object `{ ec2ParameterStore: { parameterName: "<parameter name>" }}`, for snowflake_loader_user used by loader to perform loading | `string\|map` | n/a | yes |
```

Comment:
Rather than overloading this parameter can you instead just add a new variable of `snowflake_password_from_parameter_store_arn` - if this is populated it can favor using this as the option and the config can be updated conditionally?

Reply:
Thanks for the feedback. That's what I initially did before 47c6cd2e5527823e696565203982b752b5150968, I will revert back.
It's not an ARN, so will use `snowflake_password_from_parameter_store_name`, if that's ok?

Reply:
Dropped that commit, and renamed the parameter to match the suggestion.